File: src/dataset/javascript/raindrops_exercism-javascript-oguzsh/claude/ClaudeSonnet4_raindrops_v1.js
export const convert = (number) => {
  let output = "";
  
  if (number % 3 === 0) output += "Pling";
  if (number % 5 === 0) output += "Plang";
  if (number % 7 === 0) output += "Plong";
  
  return output || String(number);
};

File: src/dataset/go/raindrops_Exercism_go_thinkverse/claude/ClaudeSonnet4_raindrops_v1.go
package raindrops

import (
	"strconv"
	"strings"
)

func Convert(number int) string {
	var parts []string

	if number%3 == 0 {
		parts = append(parts, "Pling")
	}
	if number%5 == 0 {
		parts = append(parts, "Plang")
	}
	if number%7 == 0 {
		parts = append(parts, "Plong")
	}

	if len(parts) > 0 {
		return strings.Join(parts, "")
	}

	return strconv.Itoa(number)
}
